test/launcher.spec.js

```diff
@@ -242,15 +232,15 @@ module.exports.describe = function({testRunner, expect, defaultBrowserOptions, p
     });
     it('should fire "disconnected" when closing without webSocket', async() => {
```

Comment:
This test is testing nothing - we cannot connect without webSocket anymore.

Reply:
Done